class Error(Exception):
    pass


class InternalError(Error):
    pass


class OpCallError(Error):
    pass
